src/Introspection.hack

```diff
@@ -0,0 +1,289 @@
+namespace Slack\GraphQL\Introspection;
+
+use namespace HH\Lib\Vec;
+
+interface IntrospectableSchema {
```

Comment:
If we're referencing the types in this file through the introspection namespace anyway, would it make sense to drop the `Introspectable` prefix from these names? Then they'd be referenced as `Introspection\Schema`, `Introspection\Type`, etc, which seems nice.

Reply:
Yea, good call